tests/test_tools.py

```diff
@@ -48,8 +48,8 @@ def test_coordinate_identity_transform_polar(self):
 
         A, B = po.tools.rectangular_to_polar(po.tools.polar_to_rectangular(a, b))
 
-        assert torch.norm(a - A) < 1e-3
-        assert torch.norm(b - B) < 1e-3
+        assert torch.linalg.norm(a - A) < 1e-3
+        assert torch.linalg.norm(b - B) < 1e-3
```

Comment:
```suggestion
        assert torch.linalg.vector_norm((a - A).flatten(), ord=2) < 1e-3
        assert torch.linalg.vector_norm((b - B).flatten(), ord=2) < 1e-3
```
Let's make this a bit more readable, since this is behavior that torch is doing.